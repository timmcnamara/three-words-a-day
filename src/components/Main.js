import React, { Component } from "react";
import PropTypes from "prop-types";
import Headline from "./Headline";
import SeachContainer from "./SearchContainer";
import styled from "styled-components";
import List from "./List";

const Wrapper = styled.main`
  padding: 50px;
  text-align: center;
`;

class Main extends Component {
  render() {
    return (
      <Wrapper>
        <Headline title="Make a sentence with any word..." />
        <SeachContainer />
        <List />
      </Wrapper>
    );
  }
}

export default Main;

Main.propTypes = {
  name: PropTypes.string
};
