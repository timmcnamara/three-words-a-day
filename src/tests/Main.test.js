import { render, cleanup } from "react-testing-library";
import React from "react";
import Main from "../components/Main";
import "jest-dom/extend-expect";

afterEach(cleanup);

// run a snapshot
it("render SearchContainer component", () => {
  const { asFragment } = render(<Main />);
  expect(asFragment()).toMatchSnapshot();
});
