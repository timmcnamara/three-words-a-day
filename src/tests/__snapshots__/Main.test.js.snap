// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render SearchContainer component 1`] = `
<DocumentFragment>
  <main
    class="sc-bxivhb jTcDCm"
  >
    <h1
      class="sc-bdVaJa cFuNTo"
    >
      Make a sentence with any word...
    </h1>
    <input
      aria-label="input"
      class="sc-htpNat lnVaJf"
      placeholder="Enter any word"
    />
    <button
      class="sc-bwzfXH ilHmnf"
      data-testid="button-global"
    >
      Show
    </button>
    <h1 />
  </main>
</DocumentFragment>
`;
